README.md

```diff
@@ -1,28 +1,37 @@
 # PC-Prep-Kit
-Malaria ( HTML )  project is basically developing a web application where we will spread awareness regarding Malaria by playing games. 
-All stages will be locked in starting point , and as user progresses new stages will be unlocked.
 
-THE IDEA :
-
-There there will be 4 main stages
-1. Introduction 
-2. Malaria 101 
-3. Medication & Labels 
-4. Peace Corps Policy. 
-
-All stages will be locked in starting point , and as user progresses new stages will be unlocked.
-
---Introduction : Three activities / games relating to the Introduction(These introduction activities/games  can be your creativity.)
-
---Malaria 101 :  Three activities / games relating to the disease. (These  activities/games  can be your creativity.)
-
---Medication & Labels : Three activities / games relating to medications/ diagnosis of malaria (These  activities/games  can be your creativity.)
-
---PC Policy : No activities/games , only policies of PC . 
-
-Each stage thus has three levels . As the user unlocks a stage , his progress is reported in form of fraction . For example , if a user completes one activity of the Introduction stage - It must display 1/3 for Introduction (Use creativity). As user progresses , stages will be unlocked .
-
-There is a Info-Kit section (i) at the bottom right , which will act as a repository of Knowledge/data relating to malaria .
-(One can include his/her creativity on what all can be included in it )
-
-******While designing a game/activity , keep in mind the Educational perspective of the game ********
+## What does this project do?  
+PC Prep Kit is a web application aimed at helping the peace corps volunteers with awareness about Malaria and its prevention techniques. The application uses various interactive activities to make the process effective. The App focuses on causes, prevention and medication of Malaria.
+
+## Why is the project useful? 
+PC Prep kit helps spread awareness about Malaria. According to WHO[1], In 2015 there were around 212 million cases of malaria and 429,000 deaths, 90% of which is from Africa.  It is established that early diagnosis and prompt treatment of malaria prevents deaths. Taking simple preventive measures can also help in preventing malaria to a large extent. The Peace Corps Volunteers who travel to the countries effected with malaria can use the information proved in the application and learn about symptoms, medications and their side effects and preventive measure which help keep them safe. 
+
+## How can users get started with contributing to the project? 
+The technologies used in this project are 
```

Comment:
Make this also as a title and move it upwards so that 'How can users get started with contributing to the project?' will come after that